rom/dev/src/flow/cold_reset/fmc_alias.rs

```diff
@@ -186,8 +186,8 @@ impl FmcAliasLayer {
             tcb_info_fmc_tci: &(&env.data_vault.fmc_tci()).into(),
             tcb_info_device_info_hash: &fuse_info_digest.into(),
             tcb_info_flags: &flags,
-            tcb_info_fmc_svn: &svn.to_be_bytes(),
```

Comment:
If we remove FMC SVN how do we prevent FMC rollbacks?

Reply:
For update-resets we don't allow FMC to change at all, either upgrades or downgrades.

For cold-boot resets, we prevent FMC rollbacks the same way that we prevent RT FW rollbacks: increment the SVN, which (with this change) describes the security state of both portions of the FW image. The implication is that if there's a security vulnerability identified in either FMC or RT FW, the SVN for the combined FW image would be incremented.

Reply:
There were some discussions of enabling FMC updates via the update-reset flow. I think this question was in context of that.

Reply:
If we enable FMC updates via the update-reset flow, then we'll just do the same <= check for the new firmware's FMC. In other words, we can avoid thinking about FMC and RT as two different images. They're signed together, they boot together, they update together; they have a single SVN for anti-rollback.